lib/main.js

```diff
@@ -18,6 +18,7 @@ const os = lazyReq('os');
 // Some local variables
 const errorWhitelist = [
   /^No config file found, using default configuration$/,
+  /^Using config file .+$/,
```

Comment:
This would be simpler and just as correct:
`/^Using config file /,`

Reply:
You're technically right that since this list of RegEx is tested against every line of `stderr` either version would work (and yours is slightly better for performance). The merged version works when testing against the _entire_ `stderr` if the code is refactored later to do that, and matches the behavior of the existing RegEx though.

Reply:
I committed the simplification directly on master before your reply.

Speculative benefit in case we start matching the entire stderr does not seem worthwhile.

The behaviour of the existing regex is correct because we do match the entire line, but the new regex only matches the start of the line.

So I favored the simpler code. I'll revert my change if you care.